src/features/pageObjects/pageObject.selectors.ts

```diff
@@ -4,6 +4,7 @@ import { RootState } from "../../app/store/store";
 import { locatorTaskStatus } from "../../common/constants/constants";
 import { LocatorType } from "../../common/types/common";
 import { selectClassFilterByPO } from "../filter/filter.selectors";
+import { isValidLocator } from "../locators/components/LocatorIcon";
```

Comment:
I believe that we can move `isValidLocator` in some common/utils directory

Reply:
moved to `src\features\locators\utils\utils.ts`